grails-app/controllers/org/pih/warehouse/inventory/InventoryLevelController.groovy

```diff
@@ -12,13 +12,17 @@ package org.pih.warehouse.inventory
 import grails.gorm.PagedResultList
 import grails.gorm.transactions.Transactional
 import org.pih.warehouse.core.Location
+import org.pih.warehouse.data.DataService
+import org.pih.warehouse.importer.InventoryLevelImportDataService
 import org.pih.warehouse.product.Product
+import org.pih.warehouse.product.ProductService
 
 @Transactional
 class InventoryLevelController {
 
-    def productService
-    def dataService
+    InventoryLevelImportDataService inventoryLevelImportDataService
+    ProductService productService
+    //DataService dataService
```

Comment:
if it not used, you can remove it :smile:

Reply:
Yeah I think I commented it out because as part of the refactor and still needed to see whether it was needed. Thanks for the reminder.